meanie.java

```diff
@@ -55,7 +55,7 @@ else if (weaponP.equals(""))
 	
 	//public String[][] getLocationM ()
 	//{
-		return locationM;
+		//return locationM;

```

Comment:
i dont think a String[][] for location is needed here. Kalean if you do need this just un-comment this method